playbooks/roles/postgresql_aggregation/defaults/main.yml

```diff
@@ -23,3 +23,12 @@ POSTGRESQL_AGGREGATION_PASSWORD: "password"
 POSTGRESQL_AGGREGATION_USER_ROLE: "boxes"
 POSTGRESQL_AGGREGATION_PORT: 5432
 POSTGRESQL_AGGREGATION_OPTIONS: {}
+
+postgresql_aggregation_client_cert: ''
+postgresql_aggregation_client_key: ''
+postgresql_aggregation_ca_cert: ''
+postgresql_aggregation_client_cert_path: '/etc/ssl/certs/postgresql_aggregation-client-cert.pem'
+postgresql_aggregation_client_key_path: '/etc/ssl/certs/postgresql_aggregation-client-key.pem'
+postgresql_aggregation_ca_cert_path: '/etc/ssl/certs/postgresql_aggregation-server-ca.pem'
+postgresql_aggregation_cert_owner: "{{ edxapp_user }}"
+postgresql_aggregation_cert_group: "{{ edxapp_user }}"
```

Comment:
@melvinsoft ok, that sounds fine. But then we need to make sure that this is run after the `edxapp` role, otherwise the user won't exist. I'm not sure if it's worth making edxapp a dependency for this role because I think that means the full role will be rerun.